rpcs3/rpcs3qt/update_manager.cpp

```diff
@@ -31,7 +31,9 @@
 #include <7zCrc.h>
 #include <7zFile.h>
 
+#ifndef PATH_MAX
```

Comment:
There's no guarantee the previous PATH_MAX matches MAX_PATH though. Is this intended behavior? (Potentially different length on different compilers)

Reply:
mingw-w64 defines both `PATH_MAX` and `MAX_PATH` as equal to 260. MSVC does not define it I think.